src/misc/unix/SDL_sysurl.c

```diff
@@ -47,6 +61,7 @@ int SDL_SYS_OpenURL(const char *url)
             /* Child process doesn't wait for possibly-blocking grandchild. */
             _exit(EXIT_SUCCESS);
         }
+#endif
```

Comment:
Can you add a comment ` /* USE_POSIX_SPAWN */` to the end of the `#endif` so someone glancing at this section of code knows what's happening?

Reply:
Done